configure.in

```diff
@@ -8233,7 +8239,7 @@ echo "                                       zlib               ${wxUSE_ZLIB}"
 echo "                                       expat              ${wxUSE_EXPAT}"
 echo "                                       libmspack          ${wxUSE_LIBMSPACK}"
 echo "                                       sdl                ${wxUSE_LIBSDL}"
-
+echo "                                       webview_chromium   ${wxUSE_WEBVIEW_CHROMIUM}"
```

Comment:
Ideal would be to have something like

    webview         yes (with the following backends: WebKit Chromium)

to provide maximum of information on a single line.